src/main/java/com/gluonhq/substrate/util/ProcessRunner.java

```diff
@@ -57,6 +57,7 @@
     private static boolean info;
     private static boolean logToFile;
     private final Path processLogPath;
+    private static boolean interactive;
```

Comment:
don't use static, once set it will be used by all the processes (there is a pending PR to remove the static for the other booleans too)